components/konflux-support-ops-dashboard/base/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
this component folder is not enough,  you need to create an applicationSet under argo-cd-apps and add it to one overlay, i.e. internal-staging